src/com/dotmarketing/business/UserAPIImpl.java

```diff
@@ -402,6 +403,14 @@ public void delete(User userToDelete, User replacementUser, User user, boolean r
 
 		logDelete(DeletionStage.END, userToDelete, user, "Workflows");
 
+		//replace the user reference in publishing bundles
+		logDelete(DeletionStage.BEGINNING, userToDelete, user, "Publishing Bundles");
+
+		BundleAPI bundleAPI = APILocator.getBundleAPI();

```

Comment:
we should avoid to get api reference on the code, better to use a class attribute. It is easier for testing proposes.